__tests__/app.test.js

```diff
@@ -0,0 +1,30 @@
+const request = require('supertest');
+const app = require('../app');
+const db = require('../db/connection');
+const seed = require('../db/seeds/seed');
+
+const testData = require('../db/data/test-data');
+const { categoryData } = testData;
+
+beforeEach(() => seed(testData));
+afterAll(() => db.end());
+
+describe('GET /api/categories', () => {
+  it('200: returns an array of category objects', () => {
+    return request(app)
+      .get('/api/categories')
+      .expect(200)
+      .then((response) => {
+        const { categories } = response.body;
+        expect(categories).toBeInstanceOf(Array);
+
+        expect(categories).toHaveLength(categoryData.length);
```

Comment:
categoryData.length isn't exactly robust - figure out what number the length is a test it against that.